src/main/java/org/fenixedu/academic/domain/evaluation/config/MarkSheetSettings.java

```diff
@@ -13,19 +15,20 @@ protected MarkSheetSettings() {
         setRoot(Bennu.getInstance());
     }
 
+    @Atomic
```

Comment:
The `@Atomic` annotation here isn't necessary right? 
The `create()` method is already creating a write transaction.